wsmaster/integration-tests/cascade-removal/src/test/java/org/eclipse/che/core/db/jpa/CascadeRemovalTest.java

```diff
@@ -185,7 +185,6 @@ protected void configure() {
                             SshPairImpl.class,
                             VolumeImpl.class,
                             KubernetesRuntimeState.class,
-                            KubernetesRuntimeState.RuntimeId.class,
```

Comment:
Please remove `RuntimeId` from [persistence.xml](https://github.com/garagatyi/che/blob/noEnvWS/assembly/assembly-wsmaster-war/src/main/resources/META-INF/persistence.xml#L73) as well

Reply:
Are you sure that this code is up-to-date?

Reply:
Oh, I see. Thanks!